src/import/sst_service.rs

```diff
@@ -1247,6 +1252,8 @@ mod test {
         server::raftkv,
     };
 
+    use super::ImportSstService;
+
```

Comment:
You may run `cargo fix` which automatically removes unused importing.
```suggestion
    use super::ImportSstService;

```